ci/test-add.sh

```diff
@@ -14,6 +14,18 @@ test -f pipelines/co2-emissions/run.sh
 test -f pipelines/co2-emissions/setup.sh
 test -f pipelines/co2-emissions/validate.sh
 
+init_test
+popper add popperized/swc-lesson-pipelines/co2-emissions
+test -d pipelines/co2-emissions
+test -f pipelines/co2-emissions/README.md
+popper rm co2-emissions
+
+init_test
+popper add popperized/swc-lesson-pipelines/co2-emissions swc-lesson-pipelines
```

Comment:
When a `/` is in the name, we treat the string as a path. The idea for this is to work in the same way as the `popper init` command works, that is, the second argument for the command is interpreted in the following way. If `<name>` is a string without `/`, then put it in the `pipelines/` folder. If it does contains a `/`, then treat it as a path, where the last component of the path (i.e. the `basename` of the string) is the name of the pipeline and the preceding substring is the folder.

So the following tests would have to pass:

```bash
init_test
popper add popperized/swc-lesson-pipelines/co2-emissions foo/emissions

test -d foo/emissions
cat .popper.yml | grep 'emissions:'
cat .popper.yml | grep 'path: foo/emissions'
test -f foo/emissions/run.sh

init_test
popper add popperized/swc-lesson-pipelines/co2-emissions foo

test -d pipelines/foo
cat .popper.yml | grep 'foo:'
cat .popper.yml | grep 'path: pipelines/emissions'
test -f pipelines/foo/run.sh

init_test
popper add popperized/swc-lesson-pipelines/co2-emissions foo/
test -d foo/
cat .popper.yml | grep 'foo:'
cat .popper.yml | grep 'path: foo'
test -f foo/run.sh
```